fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java

```diff
@@ -959,7 +959,7 @@ public void testSearchDate_dateTime_chained() throws Exception {
         // "dateTime" is 2019-12-31T20:00:00-04:00
 
         assertSearchReturnsComposition("subject:Basic.dateTime", "2019-12-31T20:00:00-04:00");
-        assertSearchDoesntReturnSavedResource("subject:Basic.dateTime", "2025-10-29");
+        assertSearchDoesntReturnComposition("subject:Basic.dateTime", "2025-10-29");
```

Comment:
good catch.  thanks for weeding out these latent errors I hid in here since the tests were all overridden in the one concrete subclass (fhir-persistence-jdbc)